### PR TITLE
update caption

### LINKED ISSUE #1: Add collaborators to this repo
## Comment below to be added as a collaborator

We will all be using this repository today. Even though it's public right now, you will not be able to make changes until you're given the correct permissions. We've automated this process via the GitHub API. Once you comment, we will add you as a collaborator.

You'll start to receive a lot of emails. (❗) You can slow these down immediately by clicking the **Unwatch** button at the top of the page and selecting either the **Participating and @mentions** or **Ignore** option.

### PR DESCRIPTION
**Pull request by vvaths**
closes #1

### CODE REVIEW

File: _slides/02-vvaths.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Magic Caption Here!
```
Comment (vyas000) [inline _slides/02-vvaths.md]:
```suggestion
Magic Awesome Caption Here!
```

File: _slides/02-vvaths.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (vyas000) [inline _slides/02-vvaths.md:5]: Looks great, please make the requested change

**Review by vyas000 — CHANGES_REQUESTED** [review]
looks OK.

File: _slides/02-vvaths.md (hunk 2 above)
Comment (vvaths) [reply _slides/02-vvaths.md:5]: done.

**Review by vyas000 — APPROVED** [review]
ELGTM!

**Pull request merged**